chrome/content/zotero/bindings/relatedbox.xml

```diff
@@ -279,7 +279,8 @@
 								
 								p = win.ZoteroPane;
 							}
-								
+							
+							(win || window).Zotero_Tabs.select('zotero-pane');
```

Comment:
`zotero-pane` is the library pane?

This should probably just be part of `selectItem()`? Any code that selects items (e.g., a `zotero://select` link) presumably wants to select the library pane.

Reply:
> `zotero-pane` is the library pane?

Yes.

> This should probably just be part of `selectItem()`? Any code that selects items (e.g., a `zotero://select` link) presumably wants to select the library pane.

A good point. Fixed.